tests/validate-openapi.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# This builds a rust client from the openapi spec
+
+set -ex
+
+# Ensure current path is project root
+cd "$(dirname "$0")/../"
+
+# Generate spec
+./tools/generate_openapi_models.sh
+
+# Build rust client
+docker run --rm -v "${PWD}"/openapi:/local openapitools/openapi-generator-cli generate \
+      -i /local/openapi-merged.yaml \
+      -g rust \
+      -o /local/rust-client
```

Comment:
If the problems are with docker being used here, maybe we can just download openapi generator as [jar](https://github.com/OpenAPITools/openapi-generator#13---download-jar) and run it?